libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -41,3 +41,18 @@ export const confirmedRemoveFromReadingList = createAction(
   '[Reading List API] Confirmed remove from list',
   props<{ item: ReadingListItem }>()
 );
+
+export const finishedBookFromReadingList = createAction(
+  '[Reading List API] Finish from list',
+  props<{ item: ReadingListItem }>()
+);
+
+export const failedFinishedReadingList = createAction(
+  '[Reading List API] Failed Finish from list',
+  props<{ book: Book }>()
+);
+
+export const successFinishedReadingList = createAction(
+  '[Reading List API] Confirmed Finish from list',
+  props<{ item: ReadingListItem }>()
+);
```

Comment:
Good, but I would keep more consistency in the names of the actions. Look at the other actions in this file. The pattern seems to be `actionName` - `failed<ActionName>` - `confirmed<ActionName>`. Not that important, but it's nice to keep the style of an existing code.

Reply:
Updated requested changes, please review. Thanks!